apps/speed.c

```diff
@@ -1259,30 +1257,26 @@ int speed_main(int argc, char **argv)
     int (*loopfunc)(void *args);
     loopargs_t *loopargs = NULL;
     int async_init = 0;
-    int loopargs_len = 0;
-    char *prog;
+    const char *prog;
     const char *engine_id = NULL;
     const EVP_CIPHER *evp_cipher = NULL;
     double d = 0.0;
     OPTION_CHOICE o;
     int multiblock = 0, pr_header = 0;
     int doit[ALGOR_NUM] = { 0 };
-    int ret = 1, i, k, misalign = 0;
+    int ret = 1, ix, misalign = 0;
```

Comment:
Any reason you rename i to ix?

Reply:
Fixed, I just change the signature of found() method, and it is less intrusive.